example/client/main.go

```diff
@@ -126,8 +126,8 @@ func main() {
 	glg.Infof("Finish searching %d times", testCount)
 
 	glg.Info("Start removing vector")
-	// Remove indexed 400 vectors from vald cluster.
-	for i := range ids[:insertCount] {
+	// Remove indexed 200 vectors from vald cluster.
+	for i := range ids[:insertCount/2] {
```

Comment:
Could you please define `removeCount` as a constant value like `insertCount`?
And please update the below document.
https://github.com/vdaas/vald/blob/main/docs/tutorial/get-started.md#run-example-code